src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterfacePromptForChoice.cs

```diff
@@ -412,6 +412,14 @@ private void WriteChoiceHelper(string text, ConsoleColor fg, ConsoleColor bg, re
             WriteToConsole(fg, bg, trimEnd ? text.TrimEnd(null) : text);
         }
 
+        private string ReadChoiceResponse(out ReadLineResult result)
+        {
+            result = ReadLineResult.endedOnEnter;
+            return InternalTestHooks.ForcePromptForChoiceDefaultOption
+                   ? string.Empty
+                   : ReadLine(false, "", out result, true, true);
```

Comment:
Sorry, I skipped this - please use string.Empty as in previous line.
Thank you for your patience. Seems this is my last comment.

Reply:
Changed. Not a problem.